sql/src/test/java/io/crate/operation/join/NestedLoopOperationTest.java

```diff
@@ -82,6 +83,10 @@ private Bucket executeNestedLoop(List<Row> leftRows,
         return rowReceiver.result();
     }
 
+    private static NestedLoopOperation newUnfilteredNestedLoopOperation(int phaseId, RowReceiver rowReceiver) {
```

Comment:
Maybe rename it to unfilteredInnerJoinNestedLoopOperation or at least remove the new prefix.